pkg/virt-controller/services/template.go

```diff
@@ -121,6 +121,13 @@ const EXT_LOG_VERBOSITY_THRESHOLD = 5
 
 const ephemeralStorageOverheadSize = "50M"
 
+const (
+	VirtLauncherOverhead = "150Mi" // The sum of the `ps` RSS for the 2 virt-launcher processes
+	VirtlogdOverhead     = "16Mi"  // The RSS for virtlogd
+	LibvirtdOverhead     = "33Mi"  // The RSS for libvirtd
+	QemuOverhead         = "30Mi"  // The RSS for qemu, minus the RAM of its (stressed) guest, minus the virtual page table
+)
+
```

Comment:
Do these need to be exposed?

Reply:
Exposed as in exported/uppercase? They do but only for the functional test.

Reply:
Yes, that's what I meant :)

Got you. Well, too bad golang doesn't have a way of exporting variables only to functional tests. I don't have any ideas expect for duplicating the variables, but I don't think it's better.

Alright, thanks!